Dockerfile

```diff
@@ -161,3 +167,5 @@ WORKDIR $SCANNER_HOME
 VOLUME /data
 
 ENTRYPOINT ["./scan_wrap.sh"]
+
+COPY . $SCANNER_HOME
```

Comment:
Moved the `COPY` command last, since it's non-deterministic, and Docker knows this and will always rebuild from here during `docker build`.